### PR TITLE
Category pojo

### PR DESCRIPTION
**Pull request by ColinWake**
Added category POJO with name, reviews fields, getters and a constructor.

### CODE REVIEW

**Review by ndought — APPROVED** [review]

File: src/main/java/org/wecancodeit/reviews/Review.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package org.wecancodeit.reviews;
+
+public class Review {
+
+    private String name;
+    private String description;
+    private int price;
+    private String color;
+    private double size;
```
Comment (fpemba) [inline src/main/java/org/wecancodeit/reviews/Review.java]: why do we need this?

File: src/main/java/org/wecancodeit/reviews/Review.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    public Review(String name, String description, int price, String color, double size) {
+        this.name = name;
+        this.description = description;
+        this.price = price;
+        this.color = color;
+        this.size = size;
```
Comment (fpemba) [inline src/main/java/org/wecancodeit/reviews/Review.java]: Get rid of this too

File: src/main/java/org/wecancodeit/reviews/Review.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        return price;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public double getSize() {
+        return size;
+    }
```
Comment (fpemba) [inline src/main/java/org/wecancodeit/reviews/Review.java]: this, as well.

**Review by fpemba — CHANGES_REQUESTED** [review]
fix these please.

**Review by ColinWake — COMMENTED** [review]
All outdated, fixed in newer commits.

**Review by fpemba — APPROVED** [review]
nice job.

**Pull request merged**